src/main/proto/logutil.proto

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Vitess Authors.
```

Comment:
Oh, so that's not something you defined, but it's from Vitess upstream? If so, is there any way we could pull this in as a dependency instead of copying it? If you _did_ create it yourself, can you please update the license header to use the standard Debezium one?

Reply:
Correct. All these proto files are manually copied from the master branch of Vitess VStream gRPC. They're used to generate VStream gRPC client java code. I also think it is not ideal. I just posted the question in the Vitess general slack channel as to whether there is a way to pull this as a dependency.

Reply:
Vitess does have a [Java gRPC client](https://github.com/vitessio/vitess/tree/master/java/grpc-client) and its jar is available publicly in the maven central repo [here](https://mvnrepository.com/artifact/io.vitess/vitess-grpc-client). I can try and replace the proto files with this dependency.

Reply:
@gunnarmorling Instead of generating the gRPC Java client from the proto files copied from the Vitess repo, we now use the equivalent `vitess-grpc-client` Java dependency to communicate with VStream gRPC.